### PR TITLE
Fix Readme

### PR DESCRIPTION
**Pull request by PhilKsr**
Screenshot from 2023-10-23 09-03-14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 # Essencium
 
-[![Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml/badge.svg)](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml)
+[Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml)
```
Comment (qhantom) [inline README.md:5]: This is supposed to be a status badge of our pipeline. It is corrupt due to the fact that we changed the pipeline workflow name from `build` to `ci` a couple of months ago. The correct path is:

```suggestion
[![Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml/badge.svg)]
```

File: README.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 
 [Preview](https://staging.essencium.dev)
 |
 [Docs](https://docs.essencium.dev)
 
-![Essencium Logo](./packages/app/src/img/web/icon-192.png)
+<img src="./packages/app/public/img/web/logotype_400x100px.svg" alt="Essencium Logo" width="300"/>
```
Comment (qhantom) [inline README.md:11]: You can also write this in markdown format to be consistent like the corrupt line was but with a wrong path to the logo.

**Review by qhantom — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 # Essencium
 
-[Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml)
+[![Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml/badge.svg)]
```
Comment (qhantom) [inline README.md:5]: instead of `build.yml` --> `ci.yml` at the end

**Review by qhantom — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (PhilKsr) [reply README.md]: Thought the logo is a bit huge then but changed it. :)

**Review by qhantom — APPROVED** [review]

**Pull request merged**